codespell_lib/data/dictionary.txt

```diff
@@ -11386,14 +11480,20 @@ commatas->commas, commata,
 commect->connect
 commected->connected
 commecting->connecting
+commection->connection
+commections->connections
 commectivity->connectivity
+commector->connector
+commectors->connectors
+commects->connects
 commedian->comedian
 commedians->comedians
 commedic->comedic
 commemerative->commemorative
 commemmorate->commemorate
 commemmorating->commemorating
 commen->commend, comment, common,
+commened->commented, commended, commend,
```

Comment:
Perhaps `commenced` or `communed` too, but they're probably quite rare.

Reply:
I added the other suffixes too.